prerequisites.rst

```diff
@@ -1,35 +1,250 @@
 
-* **CPU**: Any x86-64 or ARM compatible processor
-* **RAM**: 1 GiB capacity
-* **HDD**:
+Any system has software requirements and hardware requirements, also minimal
+ones and recommended ones..
 
-  * **System Drive**: min. 4 GiB capacity (plus the capacity of the RAM)
-  * **Data Drive**: capacity according to your needs
+Hardware requirements
+---------------------
 
-.. note::
-   The whole disc will be occupied by the system and swap space [1]_, so size
-   doesn't matter so much. Data storage on the system disc is not supported.
+======  ================  =========  ==========  ==================================
+ Item    supported         Minimal     Best       Recommendation
+======  ================  =========  ==========  ==================================
+ DRIV    SSD/HDD/USB...    1 disk      2 disk     2 disks: 128Gib HDD + 500Gib SSD
+ RAM     1Gib+ any          1Gib       4Gig       8Gib+ dual channel DDR4/DDR3
+ NIC     WiFi/Ether/USB     any       10Mb NIC    1GiB NIC or 10Gb NICs: SFP fiber
+ CPU     arm,x86,x64        32bit      64bit      Intel Dual Core, AMD Ryzen
+======  ================  =========  ==========  ==================================
 
-.. note::
-   Take into account in your considerations about the size of the system disk
-   that various plugins such as `openmediavault-podman` or `openmediavault-k8s`
-   store their data (e.g. container images) on the system disk.
+The storage drives (DRIV)
+^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Spinning Harddisk, SSD [2]_, Disk-on-Module [3]_, CompactFlash [4]_ or USB thumb
-drive [5]_ type drives can be used as system drive.
+This is the key of the system. |omv| supports any drive hardware and any drive 
+interface (SATA, ISE, SCSI, USB, SERIAL) but information and management will
+depend on the disk drives supported by the Debian operating system.
+
+Modern hard disks drives have firmware inside that reports several attributes, 
+Any hard disks drives is supported but |omv| only supports SMART for those
+disks that are connected to an HBA in pass-through mode.
+
+The system manages two types of storage unit classification:
+
+System drive storage
+  The Storage Disk Drive(s) used to put the system program files (by partitions), 
+  **this system drive (that is in fact a couple or more partitions) 
+  cannot officially be used for shared resources or as data drive**.
+  What supported brands and sizes are covered in the software requirements section below.
+  The recommendation for the System drive(s) hardware could be older spinning
+  Hard Disk Drives (HDD), Disk-on-Module [3]_, CompactFlash [4]_ or thumb drives (USB),
+  the Solid State Disks (SDD [2]_) must be managed.
+
+Data drive storage
+  The storage disk drive(s) where the data for the defined or used shares resources
+  wil be stored. **Cannot be the same of the system drive(s) and sizes will
+  depends of the usage of the resources**. Shared resources are managed over
```

Comment:
```suggestion
  depend of the usage of the stored user data**. Shared resources are managed over
```

Reply:
fixed in commit be624bd8da52ad54039c7533a818c8b7073b30b2